src/dmn/trn_rpc_protocol_handlers_1.c

```diff
@@ -1333,8 +1335,16 @@ int *delete_transit_network_policy_1_svc(rpc_trn_vsip_cidr_key_t *policy_key, st
 	UNUSED(rqstp);
 	static int result;
 	int rc;
-	char *itf = policy_key->interface;
-	struct vsip_cidr_t cidr;
+	char *itf = policy_key[0].interface;
```

Comment:
nit: policy_key->interface (the original way) seems more natural. Is there any reason to change style as array element?